src/components/search-button/index.tsx

```diff
@@ -76,8 +76,7 @@ const useThemeStyles = StyleSheet.createThemeHook((theme) => ({
     flexDirection: 'row',
     justifyContent: 'space-between',
     alignItems: 'center',
-    paddingHorizontal: theme.spacings.medium,
-    paddingVertical: theme.spacings.small,
+    padding: theme.spacings.medium,
```

Comment:
Just commenting on this: Changed to minHeight on the other branch, but both increased padding and defined minHeight might as well live together :)